### PR TITLE
Elixir plug ecto

### PR DESCRIPTION
**Pull request by jtwebman**
I like Phoenix but it is overkill for simple APIs when all you need is Cowboy Plug. This also includes all the tests. This replaced the elixir-cowboy tests as they were also using cowboy plug but only had the JSON and plain text tests. Also looks like the Phoenix tests are failing. I will open a different PR to fix those.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ env:
     - "TESTLANG=D"
     - "TESTLANG=Dart"
     - "TESTLANG=Dylan"
-    - "TESTLANG=Elixir"
+    - "TESTDIR=Elixir/plug"
```
Comment (nbrady-techempower) [inline .travis.yml]: Can you undo this change? Phoenix still lives in this folder and needs to be tested. The removed line will test all frameworks inside the Elixir folder so no need to update.

Comment (jtwebman) [reply .travis.yml]: The issue is the Phoenix tests are failing at least on my local they are.

Comment (nbrady-techempower) [reply .travis.yml]: I get that. Phoenix shouldn't be tested in this PR. I've written a script that only tests the frameworks of the files being touched so your PR will still pass. We need to make sure the travis.yml file covers all possible tests though as it's a master list.

Comment (jtwebman) [reply .travis.yml]: I was about to open another PR to fix those. Would you rather it just be all one PR?

Comment (nbrady-techempower) [reply .travis.yml]: That works fine for me

Comment (jtwebman) [reply .travis.yml]: Ok will switch it to the folder only.

**Comment by jtwebman** [comment]
When I switch it back my fork TravisCI only tested Phoenix. Am I missing something for it to test both?

**Comment by nbrady-techempower** [comment]
The script can be a little finicky. I'll see what it does here. On your fork are you making a PR to master or are you just testing on a branch?

**Comment by jtwebman** [comment]
Ok, no problem. I forced it to run again and it did the correct thing.
